pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go

```diff
@@ -110,7 +113,37 @@ func (c *PeriodicBackupController) sync(ctx context.Context, _ factory.SyncConte
 		}
 	}
 
-	if !defaultFound {
+	if defaultFound {
+		mirrorPods, err := c.kubeClient.CoreV1().Pods(operatorclient.TargetNamespace).List(ctx, v1.ListOptions{LabelSelector: labels.Set{"app": "etcd"}.AsSelector().String()})
```

Comment:
should we get the exact etcd pod  by pod name instead of listing the ccos-etcd pods by labelselector; or should we use the informer to cache the etcd pods info

Reply:
thanks @lance5890  for the review 

- we usually across all the code I have came across list all the pods using the selector `app=etcd` .. iiuc it is the convention across all OCP, please correct me if I'm wrong :) 
- +1 for using informers 

wdyt @tjungblu

Reply:
fixed 👍🏽